native/cocos/bindings/jswrapper/State.cpp

```diff
@@ -27,24 +27,4 @@
 #include "State.h"
 
 namespace se {
-
-State::State(Object *thisObject)
-: _thisObject(thisObject) {
-    if (_thisObject != nullptr) {
-        _thisObject->incRef();
-    }
-}
-
-State::State(Object *thisObject, const ValueArray &args)
-: _thisObject(thisObject),
-  _args(&args) {
-    if (_thisObject != nullptr) {
-        _thisObject->incRef();
-    }
-}
-
-State::~State() {
-    SAFE_DEC_REF(_thisObject);
```

Comment:
Moving the code to header could make compiled code smaller?

Reply:
To inline the destructor and code size is bearly not effected.